qcore/im.py

```diff
@@ -30,20 +30,30 @@ def order_im_cols_file(filename):
     return order_im_cols_df(df)
 
 
-def order_im_cols_df(df, pattern_order=default_pattern_order):
-    """Orders the columns in the dataframe as per the pattern order given.
+def order_im_cols_df(df, pattern_order=DEFAULT_PATTERN_ORDER):
+    """
+    Orders the columns in the dataframe as per the pattern order given.
+
+    All columns that don't match a pattern are just appended to the end in the
+    original order.
+    """
+
```

Comment:
empty line?

Reply:
Ow, I thought I did apply Black in PyCharm, will check again on Monday as I turn off my PCs these days, cannot access remotely hahaha